### PR TITLE
[CI] Replace colcon-mixin-name with colcon-defaults

### PR DESCRIPTION
**Pull request by aprotyas**
The former is not a valid option in action-ros-ci@v0.2.

Signed-off-by: Abrar Rahman Protyasha <aprotyas@u.rochester.edu>

### CODE REVIEW

File: .github/workflows/source.yml [hunk 1]
```diff
@@ -30,7 +30,12 @@ jobs:
           vcs-repo-file-url: |
             https://raw.githubusercontent.com/ros2/ros2/galactic/ros2.repos
             https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/ros2_control/ros2_control.repos
-          colcon-mixin-name: coverage-gcc
+          colcon-mixin-name: |
```
Comment (bijoua29) [inline .github/workflows/source.yml:33]: Shouldn't this be colcon-defaults:

Comment (aprotyas) [reply .github/workflows/source.yml]: Thank you for catching that! Fixed in e7c1844.

**Review by destogl — APPROVED** [review]

**Review by arne48 — APPROVED** [review]
With the fix of the issue mentioned by @bijoua29 this PR should be merge-able.

**Review by bmagyar — APPROVED** [review]

**Comment by bmagyar** [comment]
Thanks everyone!

**Comment by bmagyar** [comment]
@mergifyio backport to galactic please

**Comment by bmagyar** [comment]
Oh c'mon :D 

@mergifyio backport galactic

**Pull request merged**